index.js

```diff
@@ -86,8 +88,55 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
+      // PostCSS turns relative paths into absolute paths
+      file = path.relative(process.cwd(), file);
+      return minimatchList(file, globs);
+    }
+
+    function isImplicitComponent(file) {
```

Comment:
Let's move these functions into a separate module in `lib/` to keep this `index.js` file a little shorter.